ocaml/xenopsd/xc/device.ml

```diff
@@ -816,14 +816,14 @@ module Vif = struct
               50000L
             (* 50ms by default *)
           in
-          let bytes_per_interval =
-            ((kbytes_per_s ^* 1024L) ^* timeslice_us) ^/ 1000000L
+          let bits_per_interval =
+            ((kbps ^* 1000L) ^* timeslice_us) ^/ 1000000L
           in
-          if bytes_per_interval > 0L && bytes_per_interval < 0xffffffffL then
-            [("rate", sprintf "%Lu,%Lu" bytes_per_interval timeslice_us)]
+          if bits_per_interval > 0L && bits_per_interval < 0xffffffffL then
```

Comment:
Would suggest to write the constant as `0xffff_ffffL` to make it easier to parse; so this is expected to be a 32-bit value? I am somewhat surprised that the existing code is not using `>=` and `<=`.